From 994c70d31cb1d6993c6b1bfa2e4403d9aeb3a6a1 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:42:19 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(0ec67f2)..lineage-16.0-android-9.0.0_r31(52ad55b) (DO NOT
 SUBMIT)

Bill Yi (1):
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5044688 from 1f102c9e619de8c13b75efe818bb7d33d2ecf624 to
   pi-qpr2-release

Change-Id: I91772309e845a09fd624d84d1d5e35356a5cdf88

Change-Id: I8de11a93767a39f21b17418d30b24a012bdc678f
---
 res/values-az/strings.xml |  2 +-
 res/values-de/strings.xml |  2 +-
 res/values-it/strings.xml |  2 +-
 res/values-nl/strings.xml | 24 ++++++++++++------------
 res/values-pt/strings.xml |  2 +-
 5 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/res/values-az/strings.xml b/res/values-az/strings.xml
index 6e4e208..d675586 100644
--- a/res/values-az/strings.xml
+++ b/res/values-az/strings.xml
@@ -21,7 +21,7 @@
     <string name="description_dialpad_overflow" msgid="4853244877599437286">"Daha çox seçim"</string>
     <string name="description_delete_button" msgid="88733669653753441">"backspace"</string>
     <string name="description_image_button_plus" msgid="7821061347207448362">"plus"</string>
-    <string name="description_voicemail_button" msgid="6281470684653439120">"səsli məktub"</string>
+    <string name="description_voicemail_button" msgid="6281470684653439120">"Səsli məktub"</string>
     <string name="default_notification_description" msgid="6011631627871110785">"Defolt səs (<xliff:g id="DEFAULT_SOUND_TITLE">%1$s</xliff:g>)"</string>
     <string name="callFailed_userBusy" msgid="5163219086553610486">"Xətt məşğuldur"</string>
     <string name="callFailed_congestion" msgid="3645995103907418337">"Şəbəkə məşğuldur"</string>
diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index cb7043c..08e7ecf 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -30,7 +30,7 @@
     <string name="callFailed_number_unreachable" msgid="1243366438388873914">"Nummer nicht erreichbar"</string>
     <string name="callFailed_invalid_credentials" msgid="6115412108261660271">"Ungültiger Nutzername oder ungültiges Passwort"</string>
     <string name="callFailed_out_of_network" msgid="6970725831030399542">"Anruf außerhalb des Netzwerks"</string>
-    <string name="callFailed_server_error" msgid="1391421251190688379">"Serverfehler. Bitte versuche es später erneut."</string>
+    <string name="callFailed_server_error" msgid="1391421251190688379">"Serverfehler. Bitte versuche es später noch einmal."</string>
     <string name="callFailed_noSignal" msgid="1758972490451048086">"Kein Signal"</string>
     <string name="callFailed_limitExceeded" msgid="8796663077702587664">"ACM-Grenzwert überschritten"</string>
     <string name="callFailed_powerOff" msgid="9179061328562234362">"Mobilfunkverbindung aus"</string>
diff --git a/res/values-it/strings.xml b/res/values-it/strings.xml
index a9745c1..c6ef593 100644
--- a/res/values-it/strings.xml
+++ b/res/values-it/strings.xml
@@ -36,7 +36,7 @@
     <string name="callFailed_powerOff" msgid="9179061328562234362">"Segnale cellulare disattivato"</string>
     <string name="callFailed_simError" msgid="3307159523385380486">"Nessuna SIM o errore della SIM"</string>
     <string name="callFailed_outOfService" msgid="4535901975891969115">"Rete cellulare non disponibile"</string>
-    <string name="callFailed_fdn_only" msgid="1720606112619022283">"Chiamate in uscita limitate da FDN."</string>
+    <string name="callFailed_fdn_only" msgid="1720606112619022283">"Chiamate in uscita limitate dai numeri consentiti."</string>
     <string name="callFailed_dialToUssd" msgid="2433610980015734954">"Chiamata modificata in richiesta USSD"</string>
     <string name="callFailed_dialToSs" msgid="1932708241240329683">"Chiamata modificata in richiesta SS"</string>
     <string name="callFailed_dialToDial" msgid="4383070508519313539">"Modificata cambiando il numero"</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 4ed23c3..22da858 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -29,34 +29,34 @@
     <string name="callFailed_server_unreachable" msgid="4766146133909799091">"Server niet bereikbaar"</string>
     <string name="callFailed_number_unreachable" msgid="1243366438388873914">"Nummer niet bereikbaar"</string>
     <string name="callFailed_invalid_credentials" msgid="6115412108261660271">"Ongeldige gebruikersnaam of wachtwoord"</string>
-    <string name="callFailed_out_of_network" msgid="6970725831030399542">"Oproep buiten het netwerk"</string>
+    <string name="callFailed_out_of_network" msgid="6970725831030399542">"Gesprek buiten het netwerk"</string>
     <string name="callFailed_server_error" msgid="1391421251190688379">"Fout met server. Probeer het later opnieuw."</string>
     <string name="callFailed_noSignal" msgid="1758972490451048086">"Geen kiestoon"</string>
     <string name="callFailed_limitExceeded" msgid="8796663077702587664">"ACM-limiet overschreden"</string>
     <string name="callFailed_powerOff" msgid="9179061328562234362">"Radio uit"</string>
     <string name="callFailed_simError" msgid="3307159523385380486">"Geen simkaart of fout met simkaart"</string>
     <string name="callFailed_outOfService" msgid="4535901975891969115">"Mobiel netwerk niet beschikbaar"</string>
-    <string name="callFailed_fdn_only" msgid="1720606112619022283">"Uitgaande oproepen worden beperkt door FDN."</string>
-    <string name="callFailed_dialToUssd" msgid="2433610980015734954">"Oproep gewijzigd in USSD-verzoek"</string>
-    <string name="callFailed_dialToSs" msgid="1932708241240329683">"Oproep gewijzigd in SS-verzoek"</string>
+    <string name="callFailed_fdn_only" msgid="1720606112619022283">"Uitgaande gesprekken worden beperkt door FDN."</string>
+    <string name="callFailed_dialToUssd" msgid="2433610980015734954">"Gesprek gewijzigd in USSD-verzoek"</string>
+    <string name="callFailed_dialToSs" msgid="1932708241240329683">"Gesprek gewijzigd in SS-verzoek"</string>
     <string name="callFailed_dialToDial" msgid="4383070508519313539">"Gewijzigd in ander nummer"</string>
     <string name="callFailed_dialToDialVideo" msgid="5132568145494398814">"Gewijzigd in videogesprek"</string>
     <string name="callFailed_dialVideoToSs" msgid="4775276313558039477">"Videogesprek gewijzigd in SS-verzoek"</string>
     <string name="callFailed_dialVideoToUssd" msgid="3075648968404725328">"Videogesprek gewijzigd in USSD-verzoek"</string>
-    <string name="callFailed_dialVideoToDial" msgid="7758276428184272770">"Gewijzigd in normale oproep"</string>
+    <string name="callFailed_dialVideoToDial" msgid="7758276428184272770">"Gewijzigd in normaal gesprek"</string>
     <string name="callFailed_dialVideoToDialVideo" msgid="5451053686606499859">"Gewijzigd in ander nummer"</string>
-    <string name="callFailed_cb_enabled" msgid="4745550615395092407">"Uitgaande oproepen zijn niet mogelijk als de functie voor oproepen blokkeren is ingeschakeld."</string>
-    <string name="callFailed_dsac_restricted" msgid="4087941056515103695">"Alle oproepen worden beperkt door toegangsbeheer."</string>
+    <string name="callFailed_cb_enabled" msgid="4745550615395092407">"Uitgaande gesprekken zijn niet mogelijk als de functie voor gesprekken blokkeren is ingeschakeld."</string>
+    <string name="callFailed_dsac_restricted" msgid="4087941056515103695">"Alle gesprekken worden beperkt door toegangsbeheer."</string>
     <string name="callFailed_dsac_restricted_emergency" msgid="9031535611092828510">"Noodoproepen worden beperkt door toegangsbeheer."</string>
-    <string name="callFailed_dsac_restricted_normal" msgid="266947632344387216">"Normale oproepen worden beperkt door toegangsbeheer."</string>
+    <string name="callFailed_dsac_restricted_normal" msgid="266947632344387216">"Normale gesprekken worden beperkt door toegangsbeheer."</string>
     <string name="callFailed_unobtainable_number" msgid="124621797631734731">"Ongeldig getal"</string>
     <string name="incall_error_missing_voicemail_number" msgid="3651090591812220874">"Voicemailnummer onbekend."</string>
     <string name="callFailed_video_call_tty_enabled" msgid="186585701668604271">"Kan geen videogesprekken voeren wanneer TTY is ingeschakeld."</string>
-    <string name="callEnded_pulled" msgid="437630601519502587">"Oproep doorgeschakeld naar een ander apparaat."</string>
+    <string name="callEnded_pulled" msgid="437630601519502587">"Gesprek doorgeschakeld naar een ander apparaat."</string>
     <string name="ringtone_silent" msgid="249805466934178869">"Geen"</string>
     <string name="ringtone_unknown" msgid="2303463713486734520">"Onbekende beltoon"</string>
     <string name="callFailed_maximum_reached" msgid="4823790040422765056">"Alle beschikbare lijnen zijn in gebruik. Als je wilt bellen, verbreek je de verbinding in een van de gesprekken op dit apparaat of op een van je andere apparaten."</string>
-    <string name="callFailed_data_disabled" msgid="3885826754435317856">"De oproep is beëindigd omdat mobiele data zijn uitgeschakeld."</string>
-    <string name="callFailed_data_limit_reached_description" msgid="1439284832802552582">"De oproep is beëindigd omdat de limiet voor mobiele data is bereikt."</string>
-    <string name="callFailed_data_limit_reached" msgid="7045770594965160282">"Datalimiet bereikt. Oproep beëindigd."</string>
+    <string name="callFailed_data_disabled" msgid="3885826754435317856">"Het gesprek is beëindigd omdat mobiele data zijn uitgeschakeld."</string>
+    <string name="callFailed_data_limit_reached_description" msgid="1439284832802552582">"Het gesprek is beëindigd omdat de limiet voor mobiele data is bereikt."</string>
+    <string name="callFailed_data_limit_reached" msgid="7045770594965160282">"Datalimiet bereikt. Gesprek beëindigd."</string>
 </resources>
diff --git a/res/values-pt/strings.xml b/res/values-pt/strings.xml
index e9cfecd..6229e3f 100644
--- a/res/values-pt/strings.xml
+++ b/res/values-pt/strings.xml
@@ -34,7 +34,7 @@
     <string name="callFailed_noSignal" msgid="1758972490451048086">"Sem sinal"</string>
     <string name="callFailed_limitExceeded" msgid="8796663077702587664">"Limite de ACM excedido"</string>
     <string name="callFailed_powerOff" msgid="9179061328562234362">"Rádio desativado"</string>
-    <string name="callFailed_simError" msgid="3307159523385380486">"Sem SIM ou erro de SIM"</string>
+    <string name="callFailed_simError" msgid="3307159523385380486">"Sem chip ou erro de chip"</string>
     <string name="callFailed_outOfService" msgid="4535901975891969115">"Rede de celular não disponível."</string>
     <string name="callFailed_fdn_only" msgid="1720606112619022283">"As chamadas enviadas são restringidas pelo FDN."</string>
     <string name="callFailed_dialToUssd" msgid="2433610980015734954">"Chamada alterada para solicitação USSD"</string>
-- 
2.17.1

